graylog2-server/src/main/java/org/graylog/events/processor/EventDefinitionDto.java

```diff
@@ -54,7 +55,7 @@
 @JsonAutoDetect
 @JsonDeserialize(builder = EventDefinitionDto.Builder.class)
 @WithBeanGetter
-public abstract class EventDefinitionDto implements EventDefinition, ContentPackable<EventDefinitionEntity> {
+public abstract class EventDefinitionDto extends ScopedEntity implements EventDefinition, ContentPackable<EventDefinitionEntity> {
     public static final String FIELD_ID = "id";
```

Comment:
Not necessary but I think you could get rid of all the `id` related stuff now that you're extending `ScopedEntity`.

Reply:
Good point. Removed it.